cf-java-logging-support-servlet-jakarta/pom.xml

```diff
@@ -0,0 +1,194 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>cf-java-logging-support-servlet-jakarta</artifactId>
+	<packaging>jar</packaging>
+
+	<name>cf-java-logging-support-servlet-jakarta</name>
+	<parent>
+		<groupId>com.sap.hcp.cf.logging</groupId>
+		<artifactId>cf-java-logging-support-parent</artifactId>
+		<version>3.6.3</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+
+	<properties>
+		<servlet.api.version>5.0.0</servlet.api.version>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>com.sap.hcp.cf.logging</groupId>
+			<artifactId>cf-java-logging-support-servlet</artifactId>
+			<version>${project.version}</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>com.sap.hcp.cf.logging</groupId>
+			<artifactId>cf-java-logging-support-servlet</artifactId>
+			<version>${project.version}</version>
+			<classifier>tests</classifier>
+			<scope>provided</scope>
+		</dependency>
```

Comment:
Why is there no exclusion of `javax.servlet-api`, which is contained in `cf-java-logging-support-servlet`? Is this not necessary because of the `provided` scope?

Reply:
Meanwhile I think it's wrong to add both dependencies here, will remove them. Both artifacts are copied by the dependency-maven-plugin and shouldn't be maintained here.

Reply:
removed